aten/src/ATen/test/Dimname_test.cpp

```diff
@@ -31,32 +31,17 @@ TEST(DimnameTest, wildcardName) {
   Dimname wildcard = Dimname::wildcard();
   ASSERT_EQ(wildcard.type(), NameType::WILDCARD);
   ASSERT_EQ(wildcard.full_name(), Symbol::dimname("*"));
-  ASSERT_EQ(wildcard.untagged_name(), Symbol::dimname("*"));
 }
 
 TEST(DimnameTest, createNormalName) {
   auto foo = Symbol::dimname("foo");
   auto dimname = Dimname::fromSymbol(foo);
   ASSERT_EQ(dimname.type(), NameType::NORMAL);
   ASSERT_EQ(dimname.full_name(), foo);
-  ASSERT_EQ(dimname.untagged_name(), foo);
-
+  ASSERT_THROW(Dimname::fromSymbol(Symbol::dimname("inva.lid")), c10::Error);
```

Comment:
what are we throwing in this case? that `.` isn't allowed?

Reply:
We are throwing the error message that says "only alphabetical and underscore characters are allowed"